### PR TITLE
Add automated test

### PR DESCRIPTION
**Pull request by Chithrak07**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,36 +1,21 @@
 [tox]
```
Comment (dale-c-anderson) [inline tox.ini]: I think this tox.ini file should be removed (or renamed). If tox.ini is present, robertdebock's action ignores the image matrix we've specified, as you can see from the test results; they're all running against the default 18.04 image.

Comment (Chithrak07) [reply tox.ini]: Remove tox.ini

**Comment by dale-c-anderson** [comment]
@Chithrak07 can we please add a verify to this one to make sure nginx is listening on port 80?

**Comment by Chithrak07** [comment]
@dale-c-anderson : Done.

**Review by dale-c-anderson — APPROVED** [review]
:100: !! 

I never knew about `ss`. Handy!

**Pull request merged**